docker/build_scripts/build.sh

```diff
@@ -33,8 +33,17 @@ MANYLINUX2010_DEPS="glibc-devel libstdc++-devel glib2-devel libX11-devel libXext
 # Get build utilities
 source $MY_DIR/build_utils.sh
 
-# See https://unix.stackexchange.com/questions/41784/can-yum-express-a-preference-for-x86-64-over-i386-packages
-echo "multilib_policy=best" >> /etc/yum.conf
+# Prerequisite for architecture
+case $AUDITWHEEL_ARCH in
+x86_64)
+  # See https://unix.stackexchange.com/questions/41784/can-yum-express-a-preference-for-x86-64-over-i386-packages
+  echo "multilib_policy=best" >> /etc/yum.conf
+  ;;
+i686)
+  # CentOS 6 has a problem with curl certificate so disable cert validation
```

Comment:
What specific issues are you seeing ?
I can run `curl -fsSLO https://github.com/git/git/archive/v2.23.0.tar.gz` successfully without the `-k` option. (In fact I could build the full image without this option)

Reply:
It was throwing `curl: (77) Problem with the SSL CA cert (path? access rights?)` error without the option but it seems that it just started working well without it and I'm going to remove it.